main.py

```diff
@@ -0,0 +1,81 @@
+from kivy.app import App
+from kivy.uix.screenmanager import ScreenManager, Screen
+from kivy.core.clipboard import Clipboard
+from kivy.lang import Builder
+from filesharer import FileSharer
+import webbrowser
+import time
+
+
+Builder.load_file("frontend.kv")
```

Comment:
Looks like you can create function `run_app` and put this line and last one into this function